examples/fluids/src/setupts.c

```diff
@@ -323,7 +323,6 @@ PetscErrorCode FormIJacobian_NS(TS ts, PetscReal t, Vec Q, Vec Q_dot,
     const PetscScalar *values;
     MatType mat_type;
     PetscCall(MatGetType(J_pre, &mat_type));
-    //if (strstr(mat_type, "kokkos") || strstr(mat_type, "cusparse")) mem_type = CEED_MEM_DEVICE;
```

Comment:
That actually needs to be uncommented, not removed. It was removed during some debugging.

Reply:
https://phypid.zulipchat.com/#narrow/stream/232375-ceed-phasta/topic/MPI.20after.20Jacobian.20merge/near/285182915